.github/workflows/build_pods.yaml

```diff
@@ -0,0 +1,21 @@
+name: Build Pods
+on:
+  repository_dispatch:
+    types:
+    - build
+jobs:
+  build_pods:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: ManageIQ/manageiq-pods
```

Comment:
This shouldn't be necessary as it's the implied default (and probably makes forks harder)

Reply:
Done